testplans/lotus-soup/deals_e2e.go

```diff
@@ -75,7 +75,9 @@ func dealsE2E(t *testkit.TestEnvironment) error {
 	// give some time to the miner, otherwise, we get errors like:
 	// deal errored deal failed: (State=26) error calling node: publishing deal: GasEstimateMessageGas
 	// error: estimating gas used: message execution failed: exit 19, reason: failed to lock balance: failed to lock client funds: not enough balance to lock for addr t0102: escrow balance 0 < locked 0 + required 640297000 (RetCode=19)
-	time.Sleep(50 * time.Second)
+	time.Sleep(40 * time.Second)
+
+	time.Sleep(time.Duration(t.GlobalSeq) * 5 * time.Second)
```

Comment:
The idea here is that we give 5sec. in-between of each deal from various clients, which is certainly more than 1 block time.